### PR TITLE
Updates terrafmt to 0.3.0 + function parameter fix

### PR DESCRIPTION
**Pull request by gdavison**
Updates `terrafmt` to v0.3.0 + function parameter fix katbyte/terrafmt#41

### CODE REVIEW

File: tools/go.mod [hunk 1]
```diff
@@ -7,11 +7,11 @@ require (
...
 	github.com/golangci/golangci-lint v1.35.2
 	github.com/hashicorp/go-changelog v0.0.0-20201005170154-56335215ce3a
-	github.com/katbyte/terrafmt v0.2.1-0.20200913185704-5ff4421407b4
+	github.com/katbyte/terrafmt v0.3.0
 	github.com/pavius/impi v0.0.3 // indirect
 	github.com/terraform-linters/tflint v0.20.3
 )
 
-replace github.com/katbyte/terrafmt => github.com/gdavison/terrafmt v0.2.1-0.20201026181004-a896893cd6af
+replace github.com/katbyte/terrafmt => github.com/gdavison/terrafmt v0.3.1-0.20210204054728-84242796be99
```
Comment (bflad) [inline tools/go.mod:15]: Is it possible to get off this fork yet?

Comment (gdavison) [reply tools/go.mod:15]: Soon, hopefully. It's using an actual Go parser instead of string matching to extract the Terraform blocks.

Comment (bflad) [reply tools/go.mod:15]: Okay -- for whatever reason dependabot isn't happy with this type of Go module replacement setup.

**Review by bflad — APPROVED** [review]
LGTM once rebased and/or tidied again 🚀

**Pull request merged**